tasker/task/views.py

```diff
@@ -31,10 +32,24 @@ def task_detail(id):
 
     return render_template("task/task-detail.html", title="Task Detail", user=user, task=task)
 
-@bp.route('/add_task')
-#@login_required
+@bp.route('/add_task', methods=['GET', 'POST'])
+@login_required
 def add_task():
-    return render_template('task/add-task.html', title="Create Task")
+    form = TaskForm()
+    if form.validate_on_submit():
+        user_tz = timezone(current_user.timezone)
+        due_date = user_tz.localize(datetime.combine(form.due_date.data, datetime.min.time()))
```

Comment:
Need to add a line after 41 (and import at top `from datetime import timedelta`:
`due_date = due_date + timedelta(hours=form.hour.data)`

This will add the hours to the timestamp before creating Task.

Reply:
Added hours to due_date